docs/serverless-deployment.md

```diff
@@ -0,0 +1,67 @@
+---
+next: docs/persistence.md
+---
+
+# Serverless Deployment
+
+Serverless computing is a model which aims to abstract server management without concerns for implementing, tweaking, or scaling a server (at least, to the perspective of a user). The follow sections will provide examples on how to deploy your application to service that will run on-demand as a Function as a Service(FaaS).
```

Comment:
Can we maybe link out to some generalized resource here if folks wanna learn more about serverless? Maybe: https://serverless.com/ ?

Reply:
going to link to https://serverless.com/learn/overview